twitter.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>twitter.com/login</title>
+    <link rel="stylesheet" href="./twitter.css">
+    <link href="/YOUR_PATH/favicon.ico" rel="icon" type="image/x-icon" />
+    <link href="data:image/x-icon;base64,AAABAAEAEBAAAAEAIABoBAAAFgAAACgAAAAQAAAAIAAAAAEAIAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAOGjUjzkplOb6KhU2eyrVPXtrFX77axV4e2rVbDqqlRd56hTBgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAANygUS7doVF736JRpuSmU/HqqVT/7axV/+6sVf/urFX/7qxV/+2rVd3qqVRCAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADnp1MO7KtUqO2sVf/urFX/7qxV/+6sVf/urFX/7axV/OqpVGIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADkplMc66pUx+6sVf/urFX/7qxV/+6sVf/urFX/7qxV/+6sVf/tq1X856hTPQAAAAAAAAAAAAAAAAAAAAAAAAAA56hTx+2rVf/urFX/7qxV/+6sVf/urFX/7qxV/+6sVf/urFX/7qxV/+uqVN/kpVMJAAAAAAAAAAAAAAAA5KZTBuuqVI/trFX37qxV/+6sVf/urFX/7qxV/+6sVf/urFX/7qxV/+6sVf/tq1X/56dTZwAAAAAAAAAAAAAAAOeoU5ntq1X/7qxV/+6sVf/urFX/7qxV/+6sVf/urFX/7qxV/+6sVf/urFX/7qxV/+mpVM4AAAAAAAAAAN+iURLnp1P97atV/+6sVf/tq1X/7KtU/+qqVP/rqlT/7axV/+6sVf/urFX/7qxV/+6sVf/qqlT+4qRSEQAAAADfolET6KhUdu2rVfnrqlT/56hT/+SmU//ipFL/5KZT/+uqVP/urFX/7qxV/+6sVf/urFX/66pU/+SlUzYAAAAA4aNSBuioVOjpqVT/5KVT/9+iUdLdoVFu3KBRKeCjUi7pqVT/7axV/+6sVf/urFX/7qxV/+2rVf/np1NFAAAAAN+iUSnkpVP/4qRS9N6hUWzcoFEBAAAAAAAAAADeoVEX5KZT/uuqVP/trFX/7axV/+yrVP/qqVT/5aZT5d+iUSzcoFEX3qFR7NygUTAAAAAAAAAAAAAAAAAAAAAAAAAAAN+iUabkplP/56hT/+ioVP/lplP/4qRS/uCjUqneoVExAAAAANufUB0AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADcoFEI3aFRi9+iUe3folHx3qFRnNygURXcoFFT3KBRJgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA//8AAMD/AADAPwAA8B8AAOAPAADABwAAwAcAAIADAACAAwAAwAMAAIcDAACfAQAAvwEAAP+HAAD//wAA//8AAA==" rel="icon" type="image/x-icon" />
+</head>
+
+<body>
+    <main class="container">
+        <div class="left">
+            <div class="inputBoxes">
+                <form>
+                    <input type="text" class="login_info" placeholder="Phone, Email or username">
+                    <input type="text" class="password" placeholder="Password">
+                    <button class="logInButton" type="button">Log in</button>
+                </form>
+            </div>
+            <div class="bird">
+                <img class="birdImg" src="/Users/marvinthompson/Desktop/UNIT2/Pursuit-Core-Web-Twitter-CSS-Assignment/images/birdlogo.png" alt="Twitter logo">
```

Comment:
In order for you to link the image of the bird here you don't need to link the entire path of the file. You just need to show the path from one file in the project to another file in the same project. In this case it would look like this:
`<img class="birdImg" src="./images/birdlogo.png" alt="Twitter logo">`